### DISCUSSION TITLE
How to install mojo without root permission？

### DISCUSSION THREAD

**Discussion by 2catycm** [modular/modular#750] [Q&A]
from the guide here https://developer.modular.com/download
we have to Install modular via curl https://get.modular.com | \
  MODULAR_AUTH=mut_cbbae458f2eb491e82c5c9ef1dae9a79 \
  sh -， which requires root permission。

**Comment by mforbes** [comment]
Same question: there should be no need for a development library to require root access.  This prevents use/exploration in many research environments.

**Comment by nacho00112** [comment]
I think that's because how the Mojo binaries are built, it may need to change some things in its build system, it should do like [Zig](https://github.com/ziglang/zig).
another thing to consider in Mojo is its cross-platform capabilities, may it run one day in [Termux](https://github.com/termux/termux-app)?
